src/NServiceBus.Metrics.ServiceControl/IReportNativeQueueLength.cs

```diff
@@ -25,11 +25,14 @@ class QueueLengthBufferReporter : IReportNativeQueueLength
         TaggedLongValueWriterV1 writer;
         string[] monitoredQueues;
 
-        public QueueLengthBufferReporter(RingBuffer buffer, TaggedLongValueWriterV1 writer, params string[] monitoredQueues)
+        public QueueLengthBufferReporter(RingBuffer buffer, TaggedLongValueWriterV1 writer, ReceiveAddresses receiveAddresses)
         {
             this.buffer = buffer;
             this.writer = writer;
-            this.monitoredQueues = monitoredQueues;
+            monitoredQueues = new[]
```

Comment:
Nitpick: I would prefer to do this in the ReportingFeature to avoid having the reporter make the decision to only monitor the main input queue

Reply:
changed it to the old API, have another look